plugins/parsers/registry_test.go

```diff
@@ -44,6 +49,10 @@ func TestRegistry_BackwardCompatibility(t *testing.T) {
 	}
 
 	for name, creator := range parsers.Parsers {
+		if _, ok := notBackwardsCompatible[name]; ok {
+			t.Logf("skipping %q...", name)
+			continue
+		}
```

Comment:
Just use
```suggestion
		if name == "avro" {
			t.Logf("skipping %q...", name)
			continue
		}
```
and we can switch to a `switch` statement if more are added. This is a temporary thing anyway and backward compatibility will be removed with 2.0 anyway.